packages/scrollable-image/dev/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="zh-Hant">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0,  user-scalable=no"
+    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Test Document</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=BioRhyme"
```

Comment:
Will the google font be used in this project? If not, we can remove this `<link>` element to avoid any unexpected effect.